integration-tests/smallrye-graphql/nbactions.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file shouldn't be here

Reply:
Sorry about this - removed the file and added Netbeans files to gitignore